lua/neotest-vstest/dotnet_utils.lua

```diff
@@ -245,6 +245,31 @@ end
 
 local solution_discovery_semaphore = nio.control.semaphore(1)
 
+---@param solution_path string
+---@return string[]
+function dotnet_utils.projects(solution_path)
```

Comment:
I decided to just extract this out to a function to use it in the function `root` in the neotest interface. I was thinking about caching this, but as we discussed, it didn't really seem like there was any problems in calling this as it was pretty fast